### DISCUSSION TITLE
Mesh Messenger Missing?

### DISCUSSION THREAD

**Discussion by LVInternetworks** [amidaware/tacticalrmm#1831] [Q&A] [answered]
Probably a silly question, am I going crazy or did mesh messenger disappear? Also I cant find the ability to send alerts via dialog box any longer. Was this removed?

**Comment by wh1te909** [comment, marked as answer]
No you're not going crazy lol. There was a mesh rework but I forgot to include some perms. They're added back now in release 0.18.1 so make sure you're on the latest release, and then you will need to trigger a resync, instructions for that are [here](https://docs.tacticalrmm.com/mesh_integration/#toggle-a-full-permissions-re-sync).

**Reply by LVInternetworks** [reply]
Your a gentleman and a scholar, thank you kindly :)
